central/apitoken/datastore/telemetry.go

```diff
@@ -0,0 +1,67 @@
+package datastore
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stackrox/rox/pkg/sac/resources"
+	"github.com/stackrox/rox/pkg/search"
+	"github.com/stackrox/rox/pkg/telemetry/phonehome"
+)
+
+// Gather the number of configured API tokens.
+func Gather(ds DataStore) phonehome.GatherFunc {
+	return func(ctx context.Context) (map[string]any, error) {
+		ctx = sac.WithGlobalAccessScopeChecker(ctx,
+			sac.AllowFixedScopes(
+				sac.AccessModeScopeKeys(storage.Access_READ_ACCESS),
+				sac.ResourceScopeKeys(resources.Integration),
+			),
+		)
+		numTotalTokens, err := ds.Count(ctx, search.EmptyQuery())
+		if err != nil {
+			return nil, errors.Wrap(err, "counting all api tokens")
+		}
```

Comment:
Not sure if the following would make it any better here and below:
```go
props := map[string]any{}
_ = phonehome.AddTotal(ctx, props, "API Tokens", func(ctx context.Context) (int, error) {
	return ds.Count(ctx, search.EmptyQuery())
})
```
The returned error could probably be ignored, as `AddTotal` logs it anyway.